src/packages/server/index.js

```diff
@@ -26,9 +27,11 @@ class Server {
 
   router: Server$opts.router;
 
+  cors: Server$opts.server.cors;
```

Comment:
That's perfectly fine. The only suggestion I have is to use the `Server$config` declaration.

`cors: Server$config.cors;`